src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -76,7 +76,7 @@ export class ResultsViewPageStore {
     readonly sampleIds = remoteData(async () => {
         // first priority: user provided custom sample list
         if (this.sampleList) {
-            return this.sampleList;
+            return this.sampleList.map(sampleId => sampleId);
```

Comment:
Since `this.sampleList` is a mobx observable array, it causes problems if we return it as is. So we need to create a plain copy.